source/upgrade.txt

```diff
@@ -145,7 +145,14 @@ The v4.1 driver introduces the following breaking changes to the ABI:
 
   .. note::
 
-     This does not affect users who compile with C++17 or newer and have not set ``BSONCXX_POLY_USE_IMPLS=ON``
+     This does not affect users who compile with C++17 or newer and have not set
+     ``BSONCXX_POLY_USE_IMPLS=ON``
+
+.. _version-4.1-enable-tests-config:
+
+- Sets the ``ENABLE_TESTS`` configuration option to ``OFF`` by default for
+  auto-downloaded versions of the {+driver-short+}. :ref:`This is an expansion of the
+  change made in v4.0. <version-4.0-enable-tests-config>`
```

Comment:
I: you can move this to the "Build System Breaking Changes in v4.1" section above
Also I think the problem is with the auto-generated C version that comes with the C++ driver, and this was setting ENABLE_TESTS to ON incorrectly: 
```suggestion
- Sets the ``ENABLE_TESTS`` configuration option to ``OFF`` by default for
  auto-downloaded versions of the C driver. :ref:`This is an expansion of the
  change made in v4.0. <version-4.0-enable-tests-config>`
```